src/standalone/Vegetation/pfts.jl

```diff
@@ -0,0 +1,491 @@
+"""
+This file contains the PFT struct used to prescribe vegetation parameters to the 
+model when running using PFTs, as well as the default PFT scheme. ClimaLand 
+supports running using default PFTs, user defined PFTs, or fully prescribed 
+vegetation parameters.
+
+The default PFTs used in the model use parameter values drawn from defferent 
+sources in the literature. The sources are listed here with abbreviations that
+are used in the comments in the code below.
+
+    Chen, J. M., G. Mo, J. Pisek, J. Liu, F. Deng, M. Ishizawa, 
+    and D. Chan (2012), Effects of foliage clumping on the estimation of global
+    terrestrial gross primary productivity, Global Biogeochem. Cycles, 26, 
+    GB1019, doi:10.1029/2010GB003996.
+    abb. as Chen et al. 2012
+
+    CLM5.0 Technical Note
+    abb. as CLM5.0
+
+    CLM2 PFT Data
+    https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/pftdata/
+    abb. as CLM2 PFT Data
+"""
+
+export Pft, default_pfts, params_from_pfts
+
+using DataFrames
+
+# List of parameters that PFTs need to define in order to be valid
+param_list = [
+    # TwoStreamModel parameters
+    :Ω,          # Clumping Index
+    :α_PAR_leaf, # PAR Leaf Reflectance
+    :α_NIR_leaf, # Near-infrared Leaf Reflectance
+    :τ_PAR_leaf, # PAR Leaf Transmittance
+    :τ_NIR_leaf, # Near-infrared Leaf Transmittance
+
+    # Conductance Model
+    :g1, # Medlyn CO2 Sensitivity
+
+    # Photosynthesis model
+    :Vcmax25, # Max rate of carboxylation of Rubisco
+
+    # Plant Hydraulics and general plant parameters
+    :SAI,             # Avg Stem Area Index
```

Comment:
I believe this is site specific, not PFT specific. perhaps a PFT specific parameter is LAI/SAI? @braghiere

Reply:
There is a parameter called: 
dsladlai      = dSLA/dLAI: rate of change of SLA with LAI through canopy, projected area basis (m^2/gC)

The link to all CLM input files is here: https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/

PFTs here: https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/pftdata/
Surface data (maps) here: https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/surfdata/

Reply:
Following Land Team meeting on 2/21, leaving SAI as a site parameter and not parameterizing it in terms of PFTs for the moment. Something I didn't mention in the meeting but that I think is worth noting is that our MODIS LAI = SAI + LAI is read in automatically based on lat/long and computing an approximation for SAI from that value rather than as a PFT param can probably result in higher accuracy while still getting the value automatically as would including as a PFT value.